spec.md

```diff
@@ -387,16 +388,15 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id?acc
 }' -X PUT -H "X-Broker-API-Version: 2.11" -H "Content-Type: application/json"
 ```
 
-### Response
-
 | Status Code | Description |
-| --- | --- |
-| 201 Created | Service instance has been provisioned. The expected response body is below. |
-| 200 OK | MAY be returned if the service instance already exists and the requested parameters are identical to the existing service instance. The expected response body is below. |
-| 202 Accepted | Service instance provisioning is in progress. This triggers the platform marketplace to poll the [Last Operation](#polling-last-operation) endpoint for operation status. |
+|---|---|
+| 200 OK | MUST be returned if the service instance already exists, is fully provisioned, and the requested parameters are identical to the existing service instance. The expected response body is below. |
```

Comment:
A change from MAY to MUST feels like a spec change rather than a text fix.  It looks like the intent is that 201 can be returned in the case the service instance already existed.  Are we concerned with enforcing a change here?